spec/ndx-pose.extensions.yaml

```
@@ -91,27 +120,86 @@ groups:
       dtype: text
       doc: Version string of the software tool used.
       required: false
-  - name: nodes
+  groups:
+  - neurodata_type_inc: PoseEstimationSeries
+    doc: Estimated position data for each body part.
+    quantity: '*'
+  links:
+  - target_type: Skeleton
+    doc: Layout of body part locations and connections.
+- neurodata_type_def: TrainingFrame
+  neurodata_type_inc: NWBDataInterface
+  default_name: TrainingFrame
+  doc: Group that holds ground-truth position data for all instances in a single frame.
+  attributes:
+  - name: annotator
     dtype: text
-    dims:
-    - num_body_parts
-    shape:
-    - null
-    doc: Array of body part names corresponding to the names of the PoseEstimationSeries
-      objects within this group.
+    doc: Name of annotator who labeled the TrainingFrame.
+    required: false
+  groups:
+  - neurodata_type_inc: Instance
+    doc: Position data for a single instance in a single training frame.
+    quantity: '*'
+  - name: source_video
+    doc: Path to original video file and frame used.
```

```suggestion
  - name: source_video
    neurodata_type_inc: ImageSeries
    doc: Path to original video file and frame used.
```

An `ImageSeries` currently supports either internal or external storage (external would be preferred for this however).